buildSrc/src/main/kotlin/Toolchain.kt

```diff
@@ -66,4 +68,18 @@ fun KotlinCompile.getAccessToInternalDefinitionsOf(vararg projects: Project) {
         ).toFile()
         friendPaths.from(mainSourceSet + jarArchive)
     }
+}
+
+fun Manifest.appendMetaAttributes(project: Project) {
```

Comment:
Please move to `Publishing.kt`. 
`Toolchain.kt` is about setting-up compiler(s) toolchain.